cluster/gce/gci/configure-helper.sh

```diff
@@ -1878,10 +1887,15 @@ function start-etcd-servers {
   if [[ -e /etc/init.d/etcd ]]; then
     rm -f /etc/init.d/etcd
   fi
-  prepare-log-file /var/log/etcd.log
+  if [[ -n "${ETCD_RUNASUSER:-}" && -n "${ETCD_RUNASGROUP:-}" ]]; then
```

Comment:
The if and else can be merged into
`prepare-log-file /var/log/etcd.log ${ETCD_RUNASUSER:-0}`
`prepare-log-file /var/log/etcd-events.log ${ETCD_RUNASUSER:-0}`

Reply:
fixed.